stuffing.py

```diff
@@ -0,0 +1,147 @@
+usernames = [
```

Comment:
Is there a particular reason to refactor this file in this PR?

I find it makes the PR itself larger than necessary, I would welcome a second PR focused just around the refactor efforts and separate PR for the Authentication and Authorization work.

Reply:
I worked locally for a bunch of time before thinking "hey I should contribute it to the project". So I did a lot of work and submitted everything in one PR after reaching out to @dolevf on Twitter. It's the first time I contribute to an open source project so pardon my lack of etiquette please :)